explorer/rest/db/NemDatabase.py

```diff
@@ -0,0 +1,55 @@
+from binascii import hexlify
+
+from rest.db.DatabaseConnection import DatabaseConnection
+
+
+def _format_bytes(buffer):
+	return hexlify(buffer).decode('utf8')
+
+
+class NemDatabase(DatabaseConnection):
+	"""Database containing Nem blockchain data."""
+
+	def get_block(self, height):
+		"""Gets block by height in database."""
+
+		cursor = self.connection.cursor()
+		cursor.execute('''
+			SELECT *
+			FROM blocks
+			WHERE height = %s
+		''', (height,))
+		result = cursor.fetchone()
+
+		return {
+			'height': result[0],
+			'timestamp': result[1],
+			'total_fees': result[2],
+			'total_transactions': result[3],
+			'difficulty': result[4],
+			'block_hash': _format_bytes(result[5]),
+			'signer': _format_bytes(result[6]),
+			'signature': _format_bytes(result[7])
+		} if result else None
```

Comment:
the initial idea, I was thinking of returning a dict inside db, and `Block` is used in `NemRestFacade`.
In case some fields need to format, I can add the logic inside `NemRestFacade`.

But actually, it makes sense, to return `Block` here, all the "format" work can be done inside here.